deployment/kubernetes/agent.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       serviceAccountName: stackstate-agent
       containers:
-      - image: stackstate/stackstate-agent-2-test:master
+      - image: quay.io./stackstate/stackstate-agent-2-test:master
```

Comment:
is it `quay.io/stackstate` or `quay.io./stackstate`?

Reply:
Ah oops, good call, I'll fix that.